.github/workflows/test.yml

```diff
@@ -54,12 +53,11 @@ jobs:
     - uses: actions/checkout@v3
     - uses: ./
       with:
-        runningBranch: ${{matrix.cluster_provider}}
         cluster_provider: ${{matrix.cluster_provider}}
         bcc_version: 0.25.0
-        kubectl_version: 1.27.2
-        local_dev_cluster_version: v0.0.1
-        kind_version: 0.20.0
+        #kubectl_version: 1.27.2
```

Comment:
Can we remove the comments if it's not required?

Reply:
sure